geometry_msgs/Vector3 linear_pos 
geometry_msgs/Vector3 angular_pos
---
uint64 duration
---
float32 x_pos 
float32 y_pos
float32 theta_pos
